site/src/app/roadmap/planner/Course.tsx

```diff
@@ -48,9 +48,10 @@ export const CourseNameAndInfo: React.FC<CourseNameAndInfoProps> = (props) => {
   return (
     <PPCOverlayTrigger
       popoverContent={popoverContent}
-      placement={isMobile ? 'bottom' : openPopoverLeft ? 'left-start' : 'right-start'}
       popupListener={popupListener}
       disabled={isMobile}
+      anchor={isMobile ? 'bottom' : openPopoverLeft ? 'left' : 'right'}
+      transform={isMobile ? 'bottom' : openPopoverLeft ? 'left' : 'right'}
```

Comment:
Since we open directly to the course preview, `isMobile` no longer affects placement
```suggestion
      anchor={openPopoverLeft ? 'left' : 'right'}
      transform={openPopoverLeft ? 'left' : 'right'}
```